app/src/main/java/com/github/se/icebreakrr/model/profile/Profile.kt

```diff
@@ -39,6 +39,8 @@ data class Profile(
     }
     return age
   }
+
+  companion object
```

Comment:
I don't really understand this part, it isn't very clear

Reply:
it is actually used in my mock file to create an extension. While your current companion object doesn't have any content, it's harmless and can be useful if you plan to add shared members or utility functions for the Profile class later.